README.md

```diff
@@ -8,36 +38,39 @@ recommendations to proactively improve security by hardening the environment.
 StackRox integrates with every stage of container lifecycle: build, deploy and
 runtime.
 
-Note: the StackRox Kubernetes Security platform is built on the foundation of 
+The StackRox Kubernetes Security platform is built on the foundation of 
 the product formerly known as Prevent, which itself was called Mitigate and
 Apollo. You may find references to these previous names in code or
 documentation.
```

Comment:
Somehow this gets rendered with a large font. Maybe that's due to three dashes `---` after the text.
Did you intend to get this effect?
See here https://github.com/stackrox/stackrox/tree/mfoster-readme-update#stackrox-kubernetes-security-platform

![image](https://user-images.githubusercontent.com/537715/178540437-5cda827d-49f7-4d43-ada9-a3034f17864a.png)